dev/devicelab/bin/tasks/flutter_engine_group_performance.dart

```diff
@@ -59,6 +59,8 @@ Future<TaskResult> _doTest() async {
     final String gradlewExecutable =
         Platform.isWindows ? '.\\$gradlew' : './$gradlew';
     final String flutterPath = path.join(flutterDirectory, 'bin', 'flutter');
+    await utils.eval(flutterPath, <String>['precache', '--linux', '--android'],
```

Comment:
I don't think you need `--linux`, that is for targeting linux, not running on it.